### PR TITLE
Create laval.md

### PR DESCRIPTION
**Pull request by bktari**
ulaval_logo
Add Laval university as an academic user of Ocaml

### CODE REVIEW

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
This is missing the `academic_institution/ulaval_logo.png` file.

**Comment by bktari** [comment]
> This is missing the academic_institution/ulaval_logo.png file

How can I send you the logo file?

**Comment by sabine** [comment]
The logo needs to be added to the `data/academic_institution/` folder. Sorry, our instructions in `CONTRIBUTING.md` didn't include that.

**Comment by bktari** [comment]
> The logo needs to be added to the data/academic_institution/ folder.

Done.

File: data/academic_institutions/laval.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Laval University 
+description: >
+Laval University is a public research university located in Quebec, Quebec, Canada.
```
Comment (cuihtlauac) [inline data/academic_institutions/laval.md]:
``` suggestion
 Laval University is a public research university located in Quebec, Quebec, Canada.
```

Comment (cuihtlauac) [reply data/academic_institutions/laval.md]: Yaml syntax requires space here.

File: data/academic_institutions/ulaval_logo.png
Comment (cuihtlauac) [inline data/academic_institutions/ulaval_logo.png:1]: This file must be in the `data/media/academic_institution` folder

**Review by cuihtlauac — DISMISSED** [review]

**Review by sabine — APPROVED** [review]

**Comment by bktari** [comment]
Many thanks!

**Pull request merged**